domino/domino.py

```diff
@@ -281,11 +281,11 @@ def collaborators_add(self, usernameOrEmail, message=""):
             return disposition
 
     # App functions
-    def app_publish(self, unpublishRunningApps=True):
+    def app_publish(self, unpublishRunningApps=True, hardwareTierId=None):
         if unpublishRunningApps is True:
             self.app_unpublish()
         url = self._routes.app_publish()
-        request = {"language": "App"}
+        request = {"language": "App", "hardwareTierId": hardwareTierId}
```

Comment:
@robinkraft make sure this still works for old code that doesn't pass a hardware tier ID

Reply:
All good.